### DISCUSSION TITLE
Question!

### DISCUSSION THREAD

**Discussion by JeanClaude12345** [giswqs/geog-312#34] [Q&A] [answered]
I have a question related to earn a certificate. is this course freely to earn certificate, or it requires some payments?

**Comment by giswqs** [comment, marked as answer]
See https://geog-312.gishub.org/#enroll-for-certification

**Comment by rayhanulzannat** [comment]
Can I enroll for certification at any stage of the course?

**Reply by giswqs** [reply]
Yes

**Reply by rayhanulzannat** [reply]
Thanks! But I wanted to enrolled by using my prepaid card, but only credit card option is open, one more thing, can I have any promo code?

**Reply by giswqs** [reply]
Sorry, no promo code. The payment methods are controlled by the university IT. I don't have control over it.
